includes/Core/REST_API/REST_Routes.php

```diff
@@ -93,6 +93,40 @@ function( $server ) {
 				$this->register_routes( $server );
 			}
 		);
+
+		add_filter(
+			'do_parse_request',
+			function( $do_parse_request, $wp ) {
+				add_filter(
+					'query_vars',
+					function( $vars ) use ( $wp ) {
+						// Unsets standard public query vars to escape conflicts between WordPress core
+						// and Google Site Kit APIs which happen when WordPress incorrectly parses request
+						// arguments.
+
+						$unset_vars = ( $wp->request && stripos( $wp->request, trailingslashit( rest_get_url_prefix() ) . self::REST_ROOT ) !== false ) // Check regular permalinks.
+							|| ( empty( $wp->request ) && stripos( $this->context->input()->filter( INPUT_GET, 'rest_route' ), self::REST_ROOT ) !== false ); // Check plain permalinks.
+
+						if ( $unset_vars ) {
+							// List of variable names to remove from public query variables list.
+							return array_values(
+								array_diff(
+									$vars,
+									array(
+										'orderby',
```

Comment:
I think for now this is good, but we may consider in the future to remove almost _any_ public query var that WordPress core sets here, to avoid potential similar conflicts with other query vars. I would imagine the only public query var we'll need to still keep would be `rest_route`.

Reply:
Ok, resolving this for now.